docusaurus/docs/quickstarts/network/hosted.md

```diff
@@ -1,168 +1,235 @@
+     CGroup: /system.slice/ziti-controller.service
+             └─2375 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-controller run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/co…
+
+$ sudo systemctl -q status ziti-router --lines=0 --no-pager
+● ziti-router.service - Ziti-Router for ip-10-0-0-1-edge-router
+     Loaded: loaded (/etc/systemd/system/ziti-router.service; disabled; vendor preset: enabled)
+     Active: active (running) since Thu 2021-11-11 19:05:47 UTC; 8s ago
+   Main PID: 2385 (ziti-router)
+      Tasks: 6 (limit: 1154)
+     Memory: 231.4M
+     CGroup: /system.slice/ziti-router.service
+             └─2385 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-router run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ip-10…
+```
+
+## Adding Environment Variables Back to the Shell
+
+If you log out and log back in again you can source the '.env' file located at:
+`~/.ziti/quickstart/$(hostname -s)/$(hostname -s).env`.
+
+```bash
+# run me
+source ~/.ziti/quickstart/$(hostname -s)/$(hostname -s).env
+```
+
+```bash
+# example output
+$ source ~/.ziti/quickstart/$(hostname -s)/$(hostname -s).env
+adding /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.20.2 to the path
+
+$ echo $ZITI_HOME
+/home/ubuntu/.ziti/quickstart/ip-10-0-0-1
+```
+
+## Next Steps
+
+- [Use the Overlay](#use-the-overlay)
+- [Install Ziti Admin Console (ZAC)](#install-ziti-admin-console-zac)
+<!-- - Add a Private Router -->
+- [Add a Second Public Router](#add-a-second-public-router)
+- [Change Admin Password](#change-admin-password)
+- [Delete everything and start over](#delete-everything-and-start-over)
+
+### Use the Overlay
+
+Now you have your zero trust overlay network in place, you probably want to try it out. Head on over to
+[the services quickstart](../services/index.md) and start the journey to understanding how to use OpenZiti.
+
+### Install Ziti Admin Console (ZAC)
+
+This is an optional server app and web console for Ziti network administration.
+
+[Installation guide](../zac/installation.md)
+
+### Add a Second Public Router
+
+Routers automatically form a mesh of router links when there's more than one. In order for your express-installed router to participate in the mesh it needs a firewall exception to expose its "link listener" on port `10080/tcp`, the default port.
+
+<!-- TODO: link to the new router deployment guide when it's published -->
+
+### Change Admin Password
+
+After changing the password with `ziti` CLI, change the value of `ZITI_PWD` in `~/.ziti/quickstart/$(hostname -s)/$(hostname -s).env` to match your preferred password. This variable is used by the `zitiLogin` function.
+
+```bash
+$ zitiLogin
+Token: d6152c84-3166-4ae4-8ca3-1c38c973d450
+Saving identity 'default' to /home/ubuntu/.ziti/quickstart/ip-172-31-28-116/ziti-cli.json
+
+$ ziti edge update authenticator updb -s
+Enter your current password: 
+Enter your new password: 
+```
+
+### Delete everything and start over
```

Comment:
This is also 'noise' and not relevant to an actual user too. That's why I think this fits as a different page/link. it also uses a command I find 'scary': `rm -r "${ZITI_HOME}"` that can be messed up pretty easilyimo. the unsetZitiEnv  could use a whole lot more words too. it just doesn't add clarity to the quickstarts and is better as a reference

Reply:
I've added some words. See what you think. It feels important to maintain a procedure for starting over for reasons like: wanting to re-run with different params, something broken and user is troubleshooting.